cartography/intel/aws/ec2/instances.py

```diff
@@ -191,6 +195,15 @@ def load_ec2_instances(
                 LaunchTime=str(launch_time),
                 LaunchTimeUnix=launch_time_unix,
                 State=instance_state,
+                AvailabilityZone=instance.get("Placement").get("AvailabilityZone"),
```

Comment:
this second level field access would fail, if first level object is null. For ex: if `Placement` is null, then this row would fail. Instead, we should assign empty dictionary for first level object in this case.
Ex: `instance.get("Placement",{}).get("AvailabilityZone")`

There are few other similar cases. We should fix them as well

Reply:
Okay!